<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mqtt Subscriber client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connection</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="hostAddressLineEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
        <property name="placeholderText">
         <string>Enter host address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pingButton">
        <property name="text">
         <string>Ping</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="prefix">
         <string>Port: </string>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>1883</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="usernameEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Topic</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="topic_Lineedit">
        <property name="text">
         <string>topic/as</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="subcribeButton">
        <property name="text">
         <string>Subcribe</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Received Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Log Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="log_TextBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>hostAddressLineEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>topic_Lineedit</tabstop>
  <tabstop>subcribeButton</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>log_TextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
